packages/ffe-accordion/less/ffe-accordion.less

```diff
@@ -110,17 +110,7 @@
 }
 
 .ffe-accordion-item__content-container {
-    will-change: height;
-    height: 0;
-    overflow: hidden;
-    visibility: hidden;
     transition: height @ffe-transition-duration @ffe-ease-in-out-back;
-
-    .ffe-accordion-item--open & {
-        height: auto;
-        overflow: visible;
-        visibility: visible;
```

Comment:
As I make the PR I'm thinking: what about those who just use the styling, and not the react packages?

I suppose I should perhaps revert this? The same goes for ffe-form.less.

Reply:
Ideally yes, though I can't really say I see much involvement from the "non react" teams we still have the static pages from the CMS at sparebank1.no

Reply:
Allright, I removed the commits that removes the styling.